SignalServiceKit/src/Storage/OWSStorage.m

```diff
@@ -434,6 +443,7 @@ - (nullable id)registeredExtension:(NSString *)extensionName
 + (void)deleteDatabaseFiles
 {
     [OWSFileSystem deleteFile:[TSStorageManager databaseFilePath]];
+    [OWSFileSystem deleteFile:[OWSSessionStorage databaseFilePath]];
```

Comment:
Do we also need to delete the shm and wal files?

Reply:
Good catch, done.